src/devices/StUsb4500/samples/StUsb4500.Sample.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Device.Spi;
+using System.Linq;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Usb.Samples
```

Comment:
Now we've moved to C# 9, maybe you can do a more modern example by removing the namespace, the class and the main function?